tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -904,6 +904,37 @@ func testScalableObjectMetrics(t *testing.T) {
 			}
 		}
 		assert.Equal(t, true, found)
+
+		val, ok = family["keda_internal_scale_loop_latency_seconds_bucket"]
```

Comment:
I'm not 100% sure, but I think that the name is `keda_internal_scale_loop_latency_bucket` here
e2e test has failed here 
![image](https://github.com/user-attachments/assets/ef63d0ad-d36d-4afb-ae09-09b1f007c252)

Reply:
Hmm, I see the problem with otel. There is no otel metric for this. The problem is I don't have enough expertise to create an otel histogram metric 🤔 So we have two options. Remove e2e test or make the otel metric somehow.

Reply:
I'd say that we should include it also for otel as it has support for histograms:
https://github.com/open-telemetry/opentelemetry-go/blob/fe6c67e7e9b408d7f1e05356e4ccafadf0475b10/example/prometheus/main.go#L69-L80

Reply:
you have to call the the record function instead of using an observable approach

Reply:
Aha, now I can see in the code that I tried something, but apparently it's not the right way to do it.